frontend/packages/helm-plugin/src/catalog/utils/catalog-utils.tsx

```diff
@@ -178,3 +180,12 @@ export const normalizeHelmCharts = (
     [] as CatalogItem[],
   );
 };
+
+export const helmCatalogTypeDescription = ( // add the new QS name in the link
+  <Trans ns="helm-plugin">
+    Browse for charts that help manage complex installations and upgrades. Cluster administrators
+    can customize the content made available in the catalog. Alternatively, developers can try{' '}
+    <Link to={'/quickstart?quickstart='}>this quick start</Link> to configure their own custom Helm
```

Comment:
Asked this in slack already. Do we really want to open the quick start catalog or should we open the quick start?

Reply:
@jerolimov the quick start name will be added here , which is why I have put the PR on hold

Reply:
Yes, I added a test quickstart ID. It opens the quick start but is also redirects the user from the "Helm Charts" catalog to the "Quick Starts" catalog. Serena needs to decide if that's okay or not. I would expect that the user stays on the current page and the sidebar just opens.